app/org/sagebionetworks/bridge/play/controllers/ConsentController.java

```diff
@@ -176,7 +170,7 @@ private Result giveConsentForVersion(int version, SubpopulationGuid subpopGuid)
         
```

Comment:
Going to fix this issue, then merge. Thanks for pointing out this could be done in the consent service.